frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -148,20 +148,28 @@ class TableItem extends React.Component<{
                                   top: 0,
                                   backgroundColor: '#F8F9FA',
                                   zIndex: 1,
+                                  color: '#000000',
```

Comment:
Since the two conditions are identical except for paddingLeft, maybe consider doing something similar to https://github.com/codalab/codalab-worksheets/blob/master/frontend/src/components/worksheets/items/TableItem/BundleRow.js#L192-L197, so we won't need to update two places in the future

Reply:
Take your advice and refactor the code!